Client/src/components/common/Link/index.js

```diff
@@ -5,7 +5,7 @@ import './index.css'
 
 export default function link({ title, to }) {
```

Comment:
The name of your function should be capital letter
`export default function Link`

Reply:
done

Reply:
done